simulations/vip-171/vip-171-testnet/simulations.ts

```diff
@@ -0,0 +1,168 @@
+import { mine, setBalance } from "@nomicfoundation/hardhat-network-helpers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { expectEvents, initMainnetUser } from "../../../src/utils";
+import { forking, pretendExecutingVip, testVip } from "../../../src/vip-framework";
+import { fetchVTokenStorageCore, performVTokenBasicActions, storageLayout } from "../../../src/vtokenUpgradesHelper";
+import { CORE_MARKETS, vip171Testnet } from "../../../vips/vip-171/vip-171-testnet";
+import COMPTROLLER_ABI from "./abi/COMPTROLLER.json";
+import MOCK_TOKEN_ABI from "./abi/MOCK_TOKEN_ABI.json";
+import VTOKEN_ABI from "./abi/VTOKEN_ABI.json";
+
+const NEW_VBEP20_DELEGATE_IMPL = "0xAC5CFaC96871f35f7ce4eD2b46484Db34B548b40";
```

Comment:
This deployed contract doesn't include some of the last changes included in the repo ([here](https://github.com/VenusProtocol/venus-protocol/pull/262/files)). I would redeploy and updated version

Reply:
Updated impl addr [af09fb0](https://github.com/VenusProtocol/vips/pull/67/commits/af09fb00273c1ff846a5c74d45c6c17b0d6216b9)